src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PortableInfoboxObject.java

```diff
@@ -0,0 +1,245 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mercury;
+
+import com.wikia.webdriver.common.contentpatterns.PortableInfobox;
+import com.wikia.webdriver.common.core.Assertion;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+/**
+ * @ownshership: Content West-Wing
+ */
+public class PortableInfoboxObject extends BasePageObject {
+
+  @FindBy(css = "body")
+  private WebElement bodyElement;
+  @FindBy(css = ".portable-infobox")
+  private WebElement infoboxWrapper;
+  @FindBy(css= ".pi-hero .article-image")
+  private WebElement mainImage;
+  @FindBy(css= ".portable-infobox .pi-hero-title")
+  private WebElement title;
+  @FindBy(css= ".portable-infobox .pi-title")
+  private WebElement titleSmallImage;
+  @FindBy(css= ".portable-infobox .pi-expand-button")
+  private WebElement expandButton;
+  @FindBy(css=".article-content .collapsed")
+  private WebElement infoboxIsCollapsed;
+  @FindBy(css=".tabber .article-image")
+  private WebElement imageInTabber;
+  @FindBy(css=".tabber figcaption")
+  private WebElement captionInTabber;
+  @FindBy(css=".lightbox-media")
+  private WebElement lightbox;
+  @FindBy(css=".lightbox-close-button")
+  private WebElement lightboxCloseButton;
+  @FindBy(css=".portable-infobox .article-video")
+  private WebElement video;
+  @FindBy(css=".portable-infobox .article-video figcaption")
+  private WebElement videoCaption;
+  @FindBy(css=".pi-title img")
+  private WebElement imageInTitle;
+  @FindBy(css =".portable-infobox .new")
+  private List<WebElement> internalLinksToEmptyArticle;
+  @FindBy(css = ".portable-infobox a[href*='/wiki/']")
+  private List<WebElement> internalLinks;
+  @FindBy(css =".portable-infobox .external")
+  private List<WebElement> externalLinks;
+  @FindBy(css= ".pi-item .pi-data-label")
+  private List<WebElement> dataLabels;
+  @FindBy(css =".pi-item .pi-data-value")
+  private List<WebElement> dataValues;
+  @FindBy(css = "b")
+  private List<WebElement> boldElements;
+  @FindBy(css = "i")
+  private List<WebElement> italicElements;
+  @FindBy(css = ".portable-infobox .reference")
+  private List<WebElement> references;
+  @FindBy(css = ".portable-infobox ul li")
+  private List<WebElement> unorderedLists;
+  @FindBy(css = ".portable-infobox ol li")
+  private List<WebElement> orderedLists;
+  @FindBy(css = ".pi-header")
+  private List<WebElement> headers;
+  @FindBy(css = ".portable-infobox .pi-image img")
+  private List<WebElement> images;
+
+  public PortableInfoboxObject(WebDriver driver) {
+    super(driver);
+  }
+
+  public String getExternalLinkName(int index) {
+    Assertion.assertFalse(externalLinks.isEmpty());
+    wait.forElementVisible(externalLinks.get(index));
+    String externalLinkName = externalLinks.get(index).getText();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "externalLinkName". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)